hardnet1/main.tf

```diff
@@ -249,3 +249,56 @@ resource "google_compute_instance" "node" {
     }
   }
 }
+
+
+data "google_compute_network" "default_network" {
+  name = "default"
+}
+
+
+resource "google_compute_firewall" "fw_public_node" {
+  name = "${var.resources_name}-node-public"
+  network = "${data.google_compute_network.default_network.self_link}"
+  priority = 530
+  target_tags = [ "${var.resources_name}-node" ]
+  allow {
+    protocol = "tcp"
+    ports = [ 22, 40403 ]
+  }
+}
+
+resource "google_compute_firewall" "fw_public_node_rpc" {
+  name = "${var.resources_name}-node-rpc"
+  network = "${data.google_compute_network.default_network.self_link}"
+  priority = 540
+  target_tags = [ "${var.resources_name}-node" ]
```

Comment:
This is OK for development or public network, but later, when we're not deploying to the network from our PCs, it'd be good make only the orchestrator machine privileged to use gRPC (and deploy) via `source_tags`. Before that, this https://github.com/rchain/hardening/blob/master/hardnet1/main.tf#L155 should be changed to e.g. `${var.resources_name}-orchestrator`.

Reply:
Done as well.